.circleci/config.yml

```diff
@@ -1,28 +1,39 @@
-machine:
-  xcode:
-    version: 9.0
-  environment:
-    # Dummy values, Circle won't run without a project and scheme.
-    XCODE_PROJECT: build/tangram.xcodeproj
-    XCODE_SCHEME: phony
-dependencies:
-  pre:
-    - brew install awscli cmake
-    - gem install jazzy
-checkout:
-  post:
-    - git submodule update --init
-test:
-  override:
-    - make ios TANGRAM_IOS_FRAMEWORK_SLIM=1
-    - make ios-docs
-deployment:
-  snapshots:
-    branch: master
-    commands:
+version: 2
+jobs:
+  build-ios:
+    macos:
+      xcode: "9.0"
+    environment:
+      # Dummy values, Circle won't run without a project and scheme.
+      XCODE_PROJECT: build/tangram.xcodeproj
+      XCODE_SCHEME: phony
+    steps:
+      # Check out repository with submodules.
+      - checkout
+      - run: git submodule update --init
+      # Install dependencies.
+      - run: brew install cmake
+      - run: gem install jazzy
+      # Build framework and docs.
+      - make ios TANGRAM_IOS_FRAMEWORK_SLIM=1
+      - make ios-docs
+  build-deploy-ios-snapshot:
+    macos:
+      xcode: "9.0"
+    environment:
+      # Dummy values, Circle won't run without a project and scheme.
+      XCODE_PROJECT: build/tangram.xcodeproj
+      XCODE_SCHEME: phony
+    steps:
+      # Check out repository with submodules.
+      - checkout
+      - run: git submodule update --init
+      # Install dependencies.
+      - run: brew install awscli cmake
+      - run: gem install jazzy
       # Build the framework in debug mode and package it into pod.zip
-      - make clean-ios
```

Comment:
Probably make sense to leave `clean-ios` in there, just in case a previous build is cached.